modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/index.d.ts

```diff
@@ -93,51 +123,18 @@ interface IFrontendDataSetProps {
 	onBulkActionItemClick?: any;
 	overrideEmptyResultView?: boolean;
 	pagination?: {
-		deltas?: [
-			{
-				href?: string;
-				label: number;
-			}
-		];
+		deltas?: TDelta[];
 		initialDelta: number;
+		initialPageNumber: number;
```

Comment:
`initialDelta` and `initialPageNumber` can be optional.

Reply:
Thanks, @markocikos. I have added your suggested changes.